contracts/oracle/integration/test-oracle.ava.js

```diff
@@ -65,3 +65,24 @@ test("Should pass, oracle is an issuer", async (t) => {
   const is_used_identity = await claimer.call(oracle_contract, "is_used_identity", { 'external_id': external_id}, { gas: 200000000000000 });
   t.true(is_used_identity);
 });
+
+test("Should pass, mint sbt token and revoke (burn)", async (t) => {
+  const { registry_contract, oracle_contract, admin, claimer } = t.context.accounts;
+  const add_issuer_result = await admin.call(registry_contract, "admin_add_sbt_issuer", {'issuer': oracle_contract.accountId});
+  t.is(add_issuer_result, true);
+  let supply_by_issuer = await oracle_contract.call(registry_contract, "sbt_supply", {'issuer': oracle_contract.accountId}, { gas: 200000000000000});
+  t.assert(supply_by_issuer === 0);
+  const mint_result =  await claimer.call(oracle_contract, "sbt_mint",
+    { 'claim_b64': claim_b64,
+      'claim_sig' : sig_b64 },
+    { attachedDeposit: NEAR.parse("0.008 N").toString() },
+    { gas: Gas.parse('20 Tgas') }).catch((error) => { console.log('Transaction error:', error);});
+  t.not(mint_result, undefined);
+  const is_used_identity = await claimer.call(oracle_contract, "is_used_identity", { 'external_id': external_id}, { gas: 200000000000000 });
```

Comment:
let's use 20TGas, not 200TGas.
There is a helper method to parse `20 TGgas` to a number in `near-sdk-js`. Or we can create a global variable `const 20tgas = ....` 
Please update other places as well.

Reply:
DONE